pkg/pmem-csi-driver/nodeserver.go

```diff
@@ -289,8 +290,7 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 		// Only if created filesytem and it was XFS:
 		// Tune XFS file system to serve huge pages:
 		// Set file system extent size to 2 MiB sized and aligned block allocations.
-		_, err := pmemexec.RunCommand(ctx, "xfs_io", "-c", "extsize 2m", hostMount)
-		if err != nil {
+		if err := xfs.SetXFSExtsize(hostMount); err != nil {
```

Comment:
Here, it seems awkward that comment says "set to 2M", the subroutine is called "set size", but where this size gets set? One would expect that caller sets size via parameter. Sure that you already considered that and reached conclusion we now only need one size so we hard code it in function, and can change it later if another sizes may become into picture. But to make it look closer to what it is doing, the function could be named SetXFSExtsize2m or something.

Reply:
What about `ConfigureFS`? The caller doesn't need to care what the function does, only that it has to be called after mkfs.xfs.

Reply:
@okartau: changed, please take another look and also see my other comment about the if check.

Reply:
I agree, doing it using ConfigureFS is good solution